### DISCUSSION TITLE
⌛ 🔗 Login Widget - Implement recaptcha support

### DISCUSSION THREAD

**Discussion by secondsky** [elementor/elementor#28822] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Currently, the Login Widget (Pro Builder) doesn't allow the integration of recaptcha.

### Describe the solution you'd like.

**Enhance the code to support recaptcha as in Elementor Forms. I see this as a security measure to prevent bots using the widget to brute force their way in easier.** 

The elementor widget is currently the only point I am aware which doesn't support recaptcha. The standard Wordpress widget does, the login screen, things like woocommerce do. Just elementor seems to sleep for years and is unable to add a few lines of code to support recaptcha.

### Additional context

This request is based on this discussion (which is abandoned by the mod team for years and no one seems to be taking the issue seriously!) https://github.com/elementor/elementor/issues/3776

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by hitsgee** [comment]
I think also the My Account Elementor widget when using WooCommerce does not also benefit from Recaptcha which is crazy on a public form.

**Comment by stanwaves** [comment]
This is a long awaited feature, I don't know why it's taken ages to implement. A security feature for that matter

**Comment by manthankanani** [comment]
Follow this method [https://github.com/orgs/elementor/discussions/28607](https://github.com/orgs/elementor/discussions/28607).

I've included method to develop custom login, signup, reset password and forgot password method.

**Comment by birchof** [comment]
Still waiting for this feature, please implement
